lib/routes/routeBackbeat.js

```diff
@@ -1450,7 +1450,7 @@ function routeIndexingAPIs(request, response, userInfo, log, callback) {
 }
 
 function routeBackbeatAPIProxy(request, response, requestContexts, log) {
-    const path = request.url.replace('/_/backbeat/api', '/_/');
+    const path = request.url.replace('/_/backbeat/api/', '/_/');
```

Comment:
the same thing is in `development/9.0`, c.f. [1] (and possibly older branches):
- should we not fix it there as well?
- why did we not have the issue before? is it a subtle behavior change in some dependency we bump'ed?

[1] https://github.com/scality/cloudserver/blame/e3d9501177c4ffced964e75ba462b1748861b9cc/lib/routes/routeBackbeat.js#L1511C38-L1511C38